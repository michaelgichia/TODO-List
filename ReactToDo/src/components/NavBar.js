import React from 'react';

export default function NavBar() {
  return (
    <header>
      <nav>
        <div className="nav-wrapper">
          <a className="brand-logo right"><i className="material-icons">today</i></a>
          <a data-activates="mobile-demo" className="button-collapse"><i className="material-icons">menu</i></a>
          <ul className="side-nav" id="mobile-demo">
            <li><a>Home</a></li>
          </ul>
        </div>
      </nav>
    </header>
  )
}
