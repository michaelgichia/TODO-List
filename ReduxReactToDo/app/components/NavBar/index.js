import React from 'react';
// import styled from 'styled-components';
import AppBar from 'material-ui/AppBar';

function NavBar() {
  return (
    <div>
			<AppBar
				title="Title"
				iconClassNameRight="muidocs-icon-navigation-expand-more"
			/>
    </div>
  );
}

NavBar.propTypes = {

};

export default NavBar;

