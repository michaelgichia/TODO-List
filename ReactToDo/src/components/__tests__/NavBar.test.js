/**
 * Testing our NavBar component
 */
import React from 'react';
import { shallow } from 'enzyme';

import NavBar from '../NavBar';

it('renders correctly', () => {
  const tree = shallow(
    <NavBar />
  );
  expect(tree).toMatchSnapshot();
});