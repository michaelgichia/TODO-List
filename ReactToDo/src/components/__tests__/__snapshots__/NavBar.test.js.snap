// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <header>
    <nav>
        <div
            className="nav-wrapper"
        >
            <a
                className="brand-logo right"
            >
                <i
                    className="material-icons"
                >
                    today
                </i>
            </a>
            <a
                className="button-collapse"
                data-activates="mobile-demo"
            >
                <i
                    className="material-icons"
                >
                    menu
                </i>
            </a>
            <ul
                className="side-nav"
                id="mobile-demo"
            >
                <li>
                    <a>
                        Home
                    </a>
                </li>
            </ul>
        </div>
    </nav>
</header>,
  "nodes": Array [
    <header>
      <nav>
            <div
                  className="nav-wrapper"
            >
                  <a
                        className="brand-logo right"
                  >
                        <i
                              className="material-icons"
                        >
                              today
                        </i>
                  </a>
                  <a
                        className="button-collapse"
                        data-activates="mobile-demo"
                  >
                        <i
                              className="material-icons"
                        >
                              menu
                        </i>
                  </a>
                  <ul
                        className="side-nav"
                        id="mobile-demo"
                  >
                        <li>
                              <a>
                                    Home
                              </a>
                        </li>
                  </ul>
            </div>
      </nav>
</header>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <NavBar />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <header>
          <nav>
                    <div
                              className="nav-wrapper"
                    >
                              <a
                                        className="brand-logo right"
                              >
                                        <i
                                                  className="material-icons"
                                        >
                                                  today
                                        </i>
                              </a>
                              <a
                                        className="button-collapse"
                                        data-activates="mobile-demo"
                              >
                                        <i
                                                  className="material-icons"
                                        >
                                                  menu
                                        </i>
                              </a>
                              <ul
                                        className="side-nav"
                                        id="mobile-demo"
                              >
                                        <li>
                                                  <a>
                                                            Home
                                                  </a>
                                        </li>
                              </ul>
                    </div>
          </nav>
</header>,
        "_debugID": 2,
        "_renderedOutput": <header>
          <nav>
                    <div
                              className="nav-wrapper"
                    >
                              <a
                                        className="brand-logo right"
                              >
                                        <i
                                                  className="material-icons"
                                        >
                                                  today
                                        </i>
                              </a>
                              <a
                                        className="button-collapse"
                                        data-activates="mobile-demo"
                              >
                                        <i
                                                  className="material-icons"
                                        >
                                                  menu
                                        </i>
                              </a>
                              <ul
                                        className="side-nav"
                                        id="mobile-demo"
                              >
                                        <li>
                                                  <a>
                                                            Home
                                                  </a>
                                        </li>
                              </ul>
                    </div>
          </nav>
</header>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <NavBar />,
}
`;
